src/network/websockets/server/index.ts

```diff
@@ -53,6 +53,25 @@ export function runWebsocketServer<AuthDetails>(params: Params<AuthDetails>) {
       socket.send(JSON.stringify(obj));
     };
 
+    let cachedMessages: Array<string> = [];
+
+    const processCache = () => {
+      if (cachedMessages.length > 0) {
+        for (var msg of cachedMessages) {
+          handleMsg(msg);
+        }
+        cachedMessages = [];
+      }
+    };
+
+    const addToCache = (msg: any) => {
+      if (cachedMessages.length > 100) {
+        console.warn("Cache Size exceeded maximum allowed length");
+        socket.terminate();
```

Comment:
Since we terminate this is not particularly meaningful, but I would return to not add the message to the cache.

```suggestion
        socket.terminate();
        return;
```